src/ape/managers/chain.py

```diff
@@ -504,6 +505,42 @@ def __getitem__(self, address: AddressType) -> ContractType:
 
         return contract_type
 
+    def get_all(
```

Comment:
I like `get_all` as a name, it seems to make sense considering we have `get()`.
But am open to alternatives! Just let me know or let me know what issues you have with this one.

Reply:
`get_multiple` or `get_many` as an alternative